src/store/index.ts

```diff
@@ -1,6 +1,15 @@
 import { configureStore } from '@reduxjs/toolkit';
+import thunk from 'redux-thunk';
 import rootReducer from './reducers';
+import { createEpicMiddleware } from 'redux-observable';
+import epics from './epics';
 
-const store = configureStore({ reducer: rootReducer });
+const epicMiddleware = createEpicMiddleware();
+const middlewares = [thunk, epicMiddleware];
+const store = configureStore({
+  reducer: rootReducer,
+  middleware: getDefaultMiddleware => getDefaultMiddleware({ serializableCheck: false }).concat(middlewares),
+});
+epicMiddleware.run(epics);
 
 export { store };
```

Comment:
Minor: Do you really need such a way of exporting? I would prefer default export if there is only one entity to export from the file. But it's only my vision, what do you think?